### PR TITLE
chore(docs): doc and readme updates

### LINKED ISSUE #295: Docs: Incorrect formating in main README
### What were you searching in the docs?

Features section reads:

> [Parameters (developer preview)](https://awslabs.github.io/aws-lambda-powertools-dotnet/core/parameters/) - provides high-level functionality to retrieve one or multiple parameter values from [AWS Systems Manager Parameter Store](https://docs.aws.amazon.com/systems-manager/latest/userguide/systems-manager-parameter-store.html){target="_blank"}, [AWS Secrets Manager](https://aws.amazon.com/secrets-manager/){target="_blank"}, or [Amazon DynamoDB](https://aws.amazon.com/dynamodb/){target="_blank"}. We also provide extensibility to bring your own providers.

### Is this related to an existing documentation section?

https://github.com/awslabs/aws-lambda-powertools-dotnet/blob/develop/README.md

### How can we improve?

```markdown
Remove the {target="_blank"}
```

### Got a suggestion in mind?

_No response_

### Acknowledgment

- [X] I understand the final update might be different from my proposed suggestion, or refused.

### LINKED ISSUE #297: Docs: Error in nuget add package commands for dotnet sample
### What were you searching in the docs?

Reading the documentation on how to use with dotnet, at the section Install, there are commands that doesn't work for dotnet tool
The doc suggest to use the command `dotnet nuget add AWS.Lambda.Powertools.Tracing` but it is invalid.
It is also wrong for the next lines, for the other packages.

### Is this related to an existing documentation section?

https://awslabs.github.io/aws-lambda-powertools-dotnet/

### How can we improve?

```markdown
The correct syntax should be:
`dotnet add package AWS.Lambda.Powertools.Tracing`
```

### Got a suggestion in mind?

_No response_

### Acknowledgment

- [X] I understand the final update might be different from my proposed suggestion, or refused.

### PR DESCRIPTION
**Pull request by sliedig**
> Please provide the issue number

Issue number:
#285 

## Summary
Updates to docs and readme addressing dotnet cli corrections, updates to idempotency location, link updates to new organisation.

### Changes

> Please provide a summary of what's being changed

### User experience

> Please share what the user experience looks like before and after this change

## Checklist

Please leave checklist items unchecked if they do not apply to your change.

* [x] [Meets tenets criteria](https://docs.powertools.aws.dev/lambda-dotnet/tenets)
* [x] I have performed a self-review of this change
* [x] Changes have been tested
* [x] Changes are documented
* [x] PR title follows [conventional commit semantics](https://github.com/aws-powertools/lambda-dotnet/blob/develop/.github/semantic.yml)

<details>
<summary>Is this a breaking change?</summary>

**RFC issue number**:

Checklist:

* [ ] Migration process documented
* [ ] Implement warnings (if it can live side by side)

</details>

## Acknowledgment

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

**Disclaimer**: We value your time and bandwidth. As such, any pull requests created on non-triaged issues might not be successful.

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -62,8 +63,7 @@ markdown_extensions:
   - pymdownx.details
   - pymdownx.snippets:
       base_path: "."
-      check_paths: true
-      restrict_base_path: false
```
Comment (sliedig) [inline mkdocs.yml:66]: This is causing local development failures

File: CODEOWNERS
Comment (hjgraca) [inline CODEOWNERS:1]: Please remove this file. We will keep the one already present in the .github folder for consistency

File: docs/utilities/idempotency.md [hunk 2]
```diff
@@ -2,6 +2,10 @@
 title: Idempotency
 description: Utility
 ---
+???+ warning
+	**This utility is currently in developer preview** and is intended strictly for feedback and testing purposes **and not for production workloads**. The version and all future versions tagged with the `-preview` suffix should be treated as not stable. Until this utility is [General Availability](https://github.com/awslabs/aws-lambda-powertools-dotnet/milestone/3) we may introduce significant breaking changes and improvements in response to customers feedback.
```
Comment (hjgraca) [inline docs/utilities/idempotency.md:6]: Update URL to https://github.com/aws-powertools/powertools-lambda-dotnet/milestone/3

File: docs/we_made_this.md [hunk 3]
```diff
@@ -7,7 +7,7 @@ description: Blog posts, tutorials, and videos about Powertools for AWS Lambda c
 
 This space is dedicated to highlight our awesome community content featuring Powertools for AWS Lambda (.NET) 🙏!
 
-!!! info "[Get your content featured here](https://github.com/aws-powertools/powertools-lambda-python/issues/new?assignees=&labels=community-content&template=share_your_work.yml&title=%5BI+Made+This%5D%3A+%3CTITLE%3E){target="_blank"}!"
+!!! info "[Get your content featured here](https://github.com/aws-powertools/aws-lambda-powertools-dotnet/issues/new?assignees=&labels=community-content&template=share_your_work.yml&title=%5BI+Made+This%5D%3A+%3CTITLE%3E){target="_blank"}!"
```
Comment (hjgraca) [inline docs/we_made_this.md:10]: Update URL to https://github.com/aws-powertools/powertools-lambda-dotnet/issues/new?assignees=&labels=community-content&template=share_your_work.yml&title=%5BI+Made+This%5D%3A+%3CTITLE%3E

**Review by hjgraca — CHANGES_REQUESTED** [review]
Delete CODEOWNERS and url changes

**Review by hjgraca — APPROVED** [review]
Pushed changes to @sliedig branch

**Pull request merged**